notebooks/Working with MONAI Auto3DSeg using MONAI Cloud API.ipynb

```diff
@@ -598,9 +598,45 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# cancel automl job and inference job if not Done. This step is required before cleaning data\n",
+    "endpoint = f\"{base_url}/experiments/{automl_experiment_id}/jobs/{automl_job_id}\"\n",
+    "response = requests.get(endpoint, headers=headers)\n",
+    "if response.json()[\"status\"] != \"Done\":\n",
```

Comment:
not Done or Error?

Reply:
My opinion is that if having job status error, the notebook can continue to run, and our CI may not be able to see errors happened. Therefore, this cell can help to raise errors since in error status, job cannot be cancelled.